src/test/scala/eventstore/SoftDeleteStreamITest.scala

```diff
@@ -51,6 +53,7 @@ class SoftDeleteStreamITest extends TestConnection {
       // Long.MaxValue = 9223372036854775807
       writeMetadata("""{"$tb":9223372036854775807,"test":"test"}""")
       appendEventToRecreate()
+      expectNoMessage(100.millis) // Give ES a bit time
```

Comment:
could you explain this?

Reply:
Just me trying fix tests due to slow ES/CI. Will probably remove or replace this with something more deterministic.